roles/haproxy/templates/haproxy.cfg.j2

```diff
@@ -35,7 +35,11 @@ global
     daemon
 
     # turn on stats unix socket
+{% if develop %}
+    stats socket /var/lib/haproxy/stats mode 600 level admin
+{% else %}
     stats socket /var/lib/haproxy/stats
```

Comment:
If it makes sense for people that know HAProxy then forget I ever commented on this.